### PR TITLE
Setup Project

### PR DESCRIPTION
**Pull request by melaniesigrid**
Thank you for reviewing my project! On this PR...

- Initialize React App
- Install Redux
- Create main files and folder structure using Ducks pattern
- Create Header component
- Add React Router and NavLinks
- Create Books component with area for books and form
- Create Categories component which is under construction

### CODE REVIEW

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @melaniesigrid,

Good job so far!
✔️ App is initialized with CRA
✔️ Use of re-usable components
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] In your README, kindly add the the **Getting Started** section and include instructions on how users can get your app locally

- [x] Also remember to include instructions on how to install packages (`npm install`) and run the app (`npm start`)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -9,9 +9,21 @@
...
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Install Node and npm on your computer.
+### Setup
+Clone this repository with git clone https://github.com/melaniesigrid/reactBookstore.git in your terminal.
+Navigate to the project directory, and run npm start.
```
Comment (Mihndim2020) [inline README.md]: - [OPTIONAL] Before running the application using the `npm start` command, you will have to first of all install dependencies using the `npm install` command. Please kindly fix that.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @melaniesigrid, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (melaniesigrid) [reply README.md]: Thank you for your suggestions! I have added the commands needed.

**Pull request merged**